Backend/Matchmaking/src/main/java/matchmaking/util/ValidationUtil.java

```diff
@@ -112,7 +112,6 @@ public static void isPrevRoundOver(Matchups matchup) {
      */
     public static void isAllPlayersMatched(List<Matchups> matchups, List<Signups> players) {
         if (matchups.size() < ((players.size() + 1) / 2)) {
```

Comment:
i feel like `matchups.size() < ((players.size() + 1) / 2)` could be clearer if its like `(players.size()/matchups.size()) != 2` ?

Reply:
My math is not mathing

Reply:
Yeah shld be fine I'll change

Reply:
ok um i just ran the tests it doesn't work since if odd num, it won't be 2 cause of int aka 3 players 2 matchups will return invalid cause 1 != 2. But if I add +1 then it will fail for 2 players since 3/1 != 2 (in the case if i add +1 to account for odd)

will be changing to ((players.size() +1 /2) != matchups.size()) instead